assets/0017-SequenceDiagram-NegotiationPhase.puml

```diff
@@ -0,0 +1,55 @@
+' DIAGRAM #########################################
+' Negotiation phase messaging
+' #################################################
+@startuml
+' #################################################
+' SETTINGS: color settings of diagram
+' #################################################
+skinparam sequence {
+	BorderColor black
+	ArrowColor black
+	ActorBorderColor black
+	LifeLineBorderColor black
+	LifeLineBackgroundColor white
+	
+	ParticipantBorderColor black
+	ParticipantBackgroundColor white
+	ParticipantFontColor black
+	
+	ActorBackgroundColor white
+	ActorFontColor black
+}
+' #################################################
+' ACTORS
+' #################################################
+actor Maker as M
+' participant "app\nMaker" as TM
+' participant "Social\nMedia" as I
+' participant "app\nTaker" as TT
+actor Taker as T
+
+' allow parallelism
+!pragma teoz true
+
+' #################################################
+' DIAGRAM
+' #################################################
+group negotiation phase
+    T->M: offer request (from invite)
```

Comment:
Could you put the information you expect in this request here to make the diagram more complete?